index.html

```diff
@@ -78,7 +78,11 @@ <h2 class="text-center">Babel transforms your JavaScript</h2>
   <div class="row featurette">
     <div class="col-md-6">
       <h2 id="es2015-and-beyond">ES2015 and beyond</h2>
-      <p>Babel has support for the latest version of JavaScript through syntax transformers. These plugins allow you to use new syntax, <strong>right now</strong> without waiting for browser support. Check out our <a href="https://babeljs.io/docs/plugins/preset-latest">Latest preset</a> to get started.</p>
+      <p>
+        Babel has support for the latest version of JavaScript through syntax transformers.
+        These <a href="https://babeljs.io/docs/plugins/">plugins</a> allow you to use new syntax, <strong>right now</strong> without waiting for browser support.
+        Check out our <a href="https://babeljs.io/docs/plugins/preset-latest">Latest preset</a> to get started.
```

Comment:
It's interesting since `our latest preset` sounds like the latest one we've made